README.md

```diff
@@ -13,20 +13,22 @@ several authentication mechanisms and dynamic configuration
 
 # Contents
 
-* [Erlang Version Support](#erlang-version-support)
-* [Building eradius](#building-eradius)
-* [Using eradius](#using-eradius)
-* [Run sample server](#run-sample-server)
-* [Metrics](#metrics)
-* [RADIUS server configuration](#radius-server-configuration)
-  * [eradius configuration example 1](#eradius-configuration-example-1)
-  * [eradius configuration example 2](#eradius-configuration-example-2)
-  * [eradius configuration example 3](#eradius-configuration-example-3)
-* [Support of failover for client](#support-of-failover-for-client)
-  * [Failover configuration](#failover-configuration)
-  * [Failover Erlang code usage](#failover-erlang-code-usage)
-* [Eradius counter aggregator](#eradius-counter-aggregator)
-* [Tables](#tables)
+- [eradius](#eradius)
```

Comment:
Please revert back `-` to `*`.

Reply:
vscode changed it, I'm reverting sorry for that

Reply:
No problem, just try revert it